src/LitTemplateElement.js

```diff
@@ -0,0 +1,51 @@
+import { StyledElement } from './StyledElement.js';
+import { html } from 'lit-html';
+import { render } from 'lit-html/lib/shady-render.js';
+export { i18n } from './util/i18n.js';
+
+class LitTemplateElement extends StyledElement {
+	constructor(options) {
+		super({
+			deferUpdate: false,
+			shadowRender: false,
+			styles: [],
+			adoptGlobalStyles: true,
+			mutationObserverOptions: {
+				childList: false,
+			},
+			...options,
+		});
+		this._template = this._options.template;
+
+		if (this._options.shadowRender) this.attachShadow({ mode: 'open' });
+	}
+
+	template() {
+		return html``;
+	}
+
+	update(options) {
+		this.renderTemplate();
+		super.update(options);
+	}
+
+	renderTemplate() {
+		const template = this._template || this.template();
+		if (typeof template === 'string') {
+			// just a plain string literal. no lit-html required
+			this.getRoot().innerHTML = `${template}`;
+		} else {
+			// render via lit-html
+			render(html` ${template} `, this.getRoot(), {
+				scopeName: this.localName,
+				eventContext: this,
+			});
+		}
+	}
+
+	getRoot() {
+		return this.shadowRoot !== null ? this.shadowRoot : this;
+	}
+}
+
+export { LitTemplateElement, html, render };
```

Comment:
@lukas-schardt @quarkus 
Do we want to keep the Lit version in here? Or should we move it to an entirely new package?

If we really want to be dependency free - it must go!

I think I would rather have the same name "TemplateElement" for all template elements but only have to change the import origin.

```javascript
import { TemplateElement, html } from '@webtides/element-js';
// vs
import { TemplateElement, html } from '@webtides/element-js-templates/lit';
``` 

The naming of the package and templates/renderers can be up for discussion of course as always 🤣

Reply:
@eddyloewen I'm not really sure. Most bundlers or build tools will be able to tree-shake it out of the bundle and an extra package would be more overhead if there is an update. Still having no dependencies would be a nice thing. @quarkus What do you think?
I think i would (for now at least) prefer to keep it in the same package.